packages/frontend/src/hooks/globalSearch/useGlobalSearch.ts

```diff
@@ -10,7 +10,7 @@ export const getSearchResults = async ({
     query: string;
 }) =>
     lightdashApi<SearchResults>({
-        url: `/projects/${projectUuid}/search/${query}`,
+        url: `/projects/${projectUuid}/search/${encodeURIComponent(query)}`,
```

Comment:
better if it was `search?query=`. `/search/somerandomquery` does not make much sense as a route but not a blocker for this PR